app/imports/templates/components/bid.js

```diff
@@ -21,7 +21,7 @@ function updateRevealedStatus(template, bid) {
     // Timeout, set revealing as false
     console.log('Timeout, set revealing as false')
     MyBids.update({ _id: bid._id }, { $set: {revealing: false} });
-  }, 15000);
+  }, 60000);
```

Comment:
Yes. "Revealing" is a transitional state: it either will timeout and you should be able to try again, or it will be revealed and therefore it will be set revealing false anyway